### DISCUSSION TITLE
Problem with the communication of the ATC dispatcher

### DISCUSSION THREAD

**Discussion by yuriks59** [flybywiresim/aircraft#4983] [General]
Friends I have after updating the simulator, there was a problem related to the work of the ATC. It is (by default) in the sound setting, as before, and during the flight, the dispatcher does not turn on and you fly in silence until you click on the channel tab. I switched the communication from the ATC to the co-pilot, so as not to be distracted, and everything worked, but now it does not work. Who also has it?

**Comment by ExampleWasTaken** [comment]
I'm not sure I understand your problem. Could you maybe describe in a few words what you expect to happen vs. what actually happens?

**Reply by yuriks59** [reply]
Previously, before the May update, communication with the dispatcher during the flight worked normally, that is, after a conversation with the ATC, there were options for requests, and now it happens that the options are not activated or do not appear at all and until you click the mouse on the channels tab in the ATC window, there is no connection. I have a feeling that the option of ATC operation with interference is configured, although I have it in the settings-by default.

**Reply by ExampleWasTaken** [reply]
That is most likely an issue with either the simulator (there would be more reports though), or hardware.

**Comment by alexpro** [comment]
I have same problem - ATC hangs until I switch channels. I am not suggesting that the problem is in this particular aircraft, but such a problem exists.
